1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_africa.txt

```diff
@@ -789,4 +789,16 @@ KEN = {
 	
 	cultures = { kikuyu }
 	capital = STATE_KENYA
-}
\ No newline at end of file
+}
+
+# French West Africa
+FWA = {
```

Comment:
French West Africa should use AOF to make it easier to transfer events by standardising tags between the V2 and V3 mods